packages/chart/src/Column.ts

```diff
@@ -306,7 +307,7 @@ export class Column extends XYAxis {
                     const dataTextEnter = dataText.enter().append("g")
                         .attr("class", "dataText")
                         .each(function (this: SVGElement, d) {
-                            context.textLocal.set(this, new Text().target(this));
+                            context.textLocal.set(this, new Text().target(this).colorStroke_default("transparent"));
```

Comment:
Why is "transparent" needed by default?

Reply:
When the parent element has the "over" class (added on hover) this style is applied to the inner text:
https://github.com/hpcc-systems/Visualization/blob/master/packages/api/src/ITooltip.css#L77